docs/operator-docs/naming-executions.md

```diff
@@ -0,0 +1,78 @@
+---
+id: naming-executions
+title: Naming Executinons
+hide_title: true
+---
+
+# Naming executions
+
+By default, Cumulus will assign a random name to workflow executions. If
+desired, though, a configurable prefix can be added to those execution names.
+
+## Naming executions triggered by rules
+
+Rules now have an optional `executionNamePrefix`. If set, any workflows
+triggered directly by that rule will have an execution name that starts with
+that prefix.
+
+## Naming executions enqueued by the QueueGranules and QueuePdrs tasks
+
+The `QueueGranules` and `QueuePdrs` tasks add executions to a queue to be run
+later. These two tasks now support an optional config property called
+`executionNamePrefix`. If specified, any executions enqueued by those tasks will
+have an execution name that begins with that prefix. The value of that prefix
+should be configured in the workflow that contains the `QueueGranules` or
+`QueuePdrs` step.
+
+In the following excerpt, the `QueueGranules` `config.executionNamePrefix`
```

Comment:
I think it might be worth spelling out that you can ensure `meta.executionNamePrefix` is set by setting `rule.meta.executionNamePrefix`. I certainly wouldn't have made that connection at first

Reply:
And in this case, it might be worth showing an example rule definition just to make it really clear

Reply:
b97bb1ae5